src/java/org/apache/cassandra/schema/TableId.java

```diff
@@ -50,7 +51,8 @@
 public class TableId implements Comparable<TableId>
 {
     public static final long MAGIC = 1956074401491665062L;
-    // TODO: should this be a TimeUUID?
```

Comment:
FYI TCM moved away from time uuid so ids are deterministic now... so this comment doesn't make sense with TCM